core/src/main/java/org/eclipse/hono/util/Constants.java

```diff
@@ -51,6 +51,11 @@
      */
     public static final String PROTOCOL_ADAPTER_TYPE_KURA= "hono-kura-mqtt";
 
+    /**
+     * The "QoS-Level" request header.
```

Comment:
It seems obvious that this is the `QoS-Level` header. The name of the constant already implies it. The JavaDoc is supposed to contain a (short) description of what it is good/used for.